pages/seed/core-concepts.mdx

```diff
@@ -24,17 +24,21 @@ While generated data might be the unsung hero of early development, it can get a
 
 Using Snaplet to seed data will typically look something like this:
 
-1. Initial setup with `npx snaplet setup`
+1. Initial setup with `npx @snaplet/seed init`
 2. Using `@snaplet/seed` to generate seed data
-3. Regenerating assets as your database changes with `npx snaplet generate`
+3. Regenerating assets as your database changes with `npx @snaplet/seed dev`
```

Comment:
We can also name this something else if we settle on something else, and I'll rename the references in the docs